extensionsGallery.json

```diff
@@ -0,0 +1,44 @@
+{
+"results": [{
+	"extensions": [
+		{
+			"extensionId": "1",
+			"extensionName": "TestExtension",
+			"disolayName": "Test Extension",
+			"shortDescription": "Test Extension short Desc",
+			"publisher": {
+				"displayName":"Microsoft",
+				"publisherId": "microsoft",
+				"publisherName":"Microsoft"
+			},
+			"versions": [
+				{
+					"version": "1.0.0",
+					"lastUpdated": "1/1/2018",
+					"assetUri": "assetUri",
+					"fallbackAssetUri": "fallbackAssetUri",
+						"files": [
+						{"assetType": "Microsoft.VisualStudio.Services.VSIXPackage"},
+						{"assetType": "Microsoft.VisualStudio.Services.Links.Source"}
+					],
+					"properties": [
+					{ "key": "Microsoft.VisualStudio.Code.Engine", "value":"*"},
+					{ "key":"Microsoft.VisualStudio.Services.Links.Source", "value": "https://github.com/Microsoft/sqlopsstudio"},
+					{ "key":"Microsoft.VisualStudio.Services.VSIXPackage", "value": ""},
+					{ "key":"Microsoft.SQLOps.DownloadPage", "value": "https://github.com/Microsoft/sqlopsstudio"}
+					]
+				}
+			],
+			"statistics": [],
+			"flags": "preview"
+		}
+	],
+	"resultMetadata": [{
+		"metadataType": "ResultCount",
+		"metadataItems": [{
+			"name": "TotalCount",
+			"count": 1
+		}]
+	}]
+}]
+}

```

Comment:
Where does it get all the info that is displayed after you click on the extension? Things like rating, download count, readme, etc. We won't have to maintain that info in this json file will we?

Reply:
I have to remove the rating and download counts from the UI. other information should be from this file. I'll verify that

---
In reply to: [168322330](https://github.com/Microsoft/sqlopsstudio/pull/689#discussion_r168322330) [](ancestors = 168322330)